docs/getting-started.md

```diff
@@ -0,0 +1,347 @@
+
+```sh
+$ kompose convert                           
+INFO Kubernetes file "frontend-service.yaml" created         
+INFO Kubernetes file "redis-master-service.yaml" created     
+INFO Kubernetes file "redis-slave-service.yaml" created      
+INFO Kubernetes file "frontend-deployment.yaml" created      
+INFO Kubernetes file "redis-master-deployment.yaml" created  
+INFO Kubernetes file "redis-slave-deployment.yaml" created 
+```
+
+Alternatively, you can convert and deploy directly to Kubernetes with `kompose up`.
+
+```sh
+$ kompose up
+We are going to create Kubernetes Deployments, Services and PersistentVolumeClaims for your Dockerized application. 
+If you need different kind of resources, use the 'kompose convert' and 'kubectl create -f' commands instead. 
+
+INFO Successfully created Service: redis          
+INFO Successfully created Service: web            
+INFO Successfully created Deployment: redis       
+INFO Successfully created Deployment: web         
+
+Your application has been deployed to Kubernetes. You can run 'kubectl get deployment,svc,pods,pvc' for details.
+```
+
+
+__Access the newly deployed service:__
+
+Now that your service has been deployed, let's access it.
+
+If you're using `minikube` you may access it via the `minikube service` command.
+
+```sh
+$ minikube service frontend
+```
+
+Otherwise, use `kubectl` to see what IP the service is using:
+
+```sh
+$ kubectl describe svc frontend
+Name:                   frontend
+Namespace:              default
+Labels:                 service=frontend
+Selector:               service=frontend
+Type:                   LoadBalancer
+IP:                     10.0.0.183
+LoadBalancer Ingress:   123.45.67.89
+Port:                   80      80/TCP
+NodePort:               80      31144/TCP
+Endpoints:              172.17.0.4:80
+Session Affinity:       None
+No events.
+
+```
+
+Note: If you're using a cloud provider, your IP will be listed next to `LoadBalancer Ingress`.
+
+If you have yet to expose your service (for example, within GCE), use the command:
+
+```sh
+kubectl expose deployment frontend --type="LoadBalancer" 
+```
+
+To check functionality, you may also `curl` the URL.
+
+```sh
+$ curl http://123.45.67.89
+```
+
+## Minishift and Kompose
+
+In this guide, we'll deploy a sample `docker-compose.yaml` file to an OpenShift cluster.
+
+Requirements:
+  - [minishift](https://github.com/minishift/minishift)
+  - [kompose](https://github.com/kubernetes/kompose)
+  - An OpenShift route created
+
+__Note:__ The service will NOT be accessible until you create an OpenShift route with `oc expose`. You must also have a virtualization environment setup. By default, `minishift` uses VirtualBox.
+
+__Start `minishift`:__
+
+[Minishift](https://github.com/minishift/minishift) is a tool that helps run OpenShift locally using a single-node cluster inside of a VM. Similar to [minikube](https://github.com/kubernetes/minikube].
```

Comment:
Broken hyperlink:
it should be
[minikube](https://github.com/kubernetes/minikube)